from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_Test_dialog(object):
    def setupUi(self, Test_dialog):
        Test_dialog.setObjectName("Test_dialog")
        Test_dialog.resize(428, 186)
        self.verticalLayout_2 = QtWidgets.QVBoxLayout(Test_dialog)
        self.verticalLayout_2.setObjectName("verticalLayout_2")
        self.label = QtWidgets.QLabel(Test_dialog)
        self.label.setTextFormat(QtCore.Qt.PlainText)
        self.label.setTextInteractionFlags(QtCore.Qt.LinksAccessibleByMouse|QtCore.Qt.TextSelectableByMouse)
        self.label.setObjectName("label")
        self.verticalLayout_2.addWidget(self.label, 0, QtCore.Qt.AlignHCenter)
        self.status_widget = QtWidgets.QWidget(Test_dialog)
        self.status_widget.setObjectName("status_widget")
        self.verticalLayout = QtWidgets.QVBoxLayout(self.status_widget)
        self.verticalLayout.setObjectName("verticalLayout")
        self.widget_2 = QtWidgets.QWidget(self.status_widget)
        self.widget_2.setObjectName("widget_2")
        self.horizontalLayout_2 = QtWidgets.QHBoxLayout(self.widget_2)
        self.horizontalLayout_2.setObjectName("horizontalLayout_2")
        self.label_2 = QtWidgets.QLabel(self.widget_2)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Maximum, QtWidgets.QSizePolicy.Preferred)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.label_2.sizePolicy().hasHeightForWidth())
        self.label_2.setSizePolicy(sizePolicy)
        self.label_2.setObjectName("label_2")
        self.horizontalLayout_2.addWidget(self.label_2)
        self.status_label = QtWidgets.QLabel(self.widget_2)
        self.status_label.setTextFormat(QtCore.Qt.PlainText)
        self.status_label.setTextInteractionFlags(QtCore.Qt.LinksAccessibleByMouse|QtCore.Qt.TextSelectableByMouse)
        self.status_label.setObjectName("status_label")
        self.horizontalLayout_2.addWidget(self.status_label)
        self.verticalLayout.addWidget(self.widget_2)
        self.verticalLayout_2.addWidget(self.status_widget)
        self.widget = QtWidgets.QWidget(Test_dialog)
        self.widget.setObjectName("widget")
        self.horizontalLayout = QtWidgets.QHBoxLayout(self.widget)
        self.horizontalLayout.setObjectName("horizontalLayout")
        self.cancel_btn = QtWidgets.QPushButton(self.widget)
        self.cancel_btn.setObjectName("cancel_btn")
        self.horizontalLayout.addWidget(self.cancel_btn)
        self.save_btn = QtWidgets.QPushButton(self.widget)
        self.save_btn.setObjectName("save_btn")
        self.horizontalLayout.addWidget(self.save_btn)
        self.verticalLayout_2.addWidget(self.widget, 0, QtCore.Qt.AlignRight)

        self.retranslateUi(Test_dialog)
        self.cancel_btn.clicked['bool'].connect(Test_dialog.close) # type: ignore
        QtCore.QMetaObject.connectSlotsByName(Test_dialog)

    def retranslateUi(self, Test_dialog):
        _translate = QtCore.QCoreApplication.translate
        Test_dialog.setWindowTitle(_translate("Test_dialog", "Connection"))
        self.label.setText(_translate("Test_dialog", "Connection Test Result"))
        self.label_2.setText(_translate("Test_dialog", "Status : "))
        self.status_label.setText(_translate("Test_dialog", "Connection Status"))
        self.cancel_btn.setText(_translate("Test_dialog", "Cancel"))
        self.save_btn.setText(_translate("Test_dialog", "Save"))
